x-pack/test_serverless/api_integration/test_suites/common/reporting/csv_v2_esql.ts

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import request from 'supertest';
+
+import { DISCOVER_APP_LOCATOR } from '@kbn/discover-plugin/common';
+import { InternalRequestHeader, RoleCredentials } from '@kbn/ftr-common-functional-services';
+import type { ReportApiJSON } from '@kbn/reporting-common/types';
+import type { JobParamsCsvFromSavedObject } from '@kbn/reporting-export-types-csv-common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ getService }: FtrProviderContext) => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+  const log = getService('log');
+  const reportingAPI = getService('svlReportingApi');
+  const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  let roleAuthc: RoleCredentials;
+  let internalReqHeader: InternalRequestHeader;
+
+  // Helper function
+  const requestCsv = async (
+    params: Omit<JobParamsCsvFromSavedObject, 'objectType' | 'browserTimezone' | 'version'>
+  ) => {
+    const job: JobParamsCsvFromSavedObject = {
+      browserTimezone: (params as JobParamsCsvFromSavedObject).browserTimezone ?? 'UTC',
+      objectType: 'search',
+      version: '8.13.0',
+      title: 'CSV Report',
+      ...params,
+    };
+    log.info(`sending request for query: ${JSON.stringify(job.locatorParams[0].params.query)}`);
+
+    return await reportingAPI.createReportJobInternal('csv_v2', job, roleAuthc, internalReqHeader);
+  };
+
+  describe('CSV Generation from ES|QL', () => {
+    describe('export from non-timebased data view', () => {
+      const timelessIndexName = 'timeless-test';
+      const loadTimelessData = async () => {
+        log.info(`loading test data`);
+        await es.indices.create({
+          index: timelessIndexName,
+          body: {
+            settings: { number_of_shards: 1 },
+            mappings: {
+              properties: {
+                eon: { type: 'keyword' },
+                era: { type: 'keyword' },
+                period: { type: 'keyword' },
+                epoch: { type: 'keyword' },
+              },
+            },
+          },
+        });
+        await es.bulk({
+          refresh: 'wait_for',
+          body: [
+            { index: { _index: timelessIndexName, _id: 'tvJJX4UBvD7uFsw9L2x4' } },
```

Comment:
What is the benefit of using those complex `_id`? Instead of `'1', '2', '3'...`
I looks like they are all the same but actually they are not and I am not sure if we are trying to test `_id` overriding here

Reply:
Good point. I copied this from the stateful tests and it looks like someone grabbed a section of search results and formatted it. I will simplify.